### PR TITLE
QG5 changes

### PR DESCRIPTION
**Pull request by adkumar1**

### CODE REVIEW

**Review by carslen — CHANGES_REQUESTED** [review]
Good job, we're making progress! 

Please: 
- add missing `charts/SDFactory/Readme.md` file. You can use [helm-docs](https://github.com/norwoodj/helm-docs) to generate it. This is a good time to add valuable comments to your `values.yaml` file, as helm-docs will add the comment to the description field.
- resolve the requested changes (refer to comments), there are a few

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ## [1.2.0] - 2023-02-23
 QG5 changes
```
[the remark this answers is not in the document]
Comment (adkumar1) [reply CHANGELOG.md]: Fixed it.

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,6 +15,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
 
+## [1.2.1] - 2023-02-16
```
[the remark this answers is not in the document]
Comment (adkumar1) [reply CHANGELOG.md:15]: Fixed it.

File: INSTALL.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+How to install application using helm:-
+    helm install ReleaseName ChartName
+    
+    a.) Add helm repository in tractusx:-
+           helm repo add sd-factory https://eclipse-tractusx.github.io/charts/dev
+    b.) To search the specific repo in helm repositories 
+           helm search repo tractusx-dev
+    c.) To install using helm command:-   
+           helm install sd-factory tractusx-dev/sd-factory
+
```
[the remark this answers is not in the document]
Comment (adkumar1) [reply INSTALL.md:16]: There are no dependencies in this application. so we are good

File: README.md [hunk 4]
```diff
@@ -252,13 +253,26 @@ the images need to be created as it is [described here](#docker). Do not forget
...
 and the Custodian Wallet.
 
+
+
 ## Installation Steps:-
 
 Helm charts are provided inside https://github.com/eclipse-tractusx/sd-factory
 
-There are diffrent ways to do the installation
+1.) Using helm commands:- <br />
```
[the remark this answers is not in the document]
Comment (adkumar1) [reply README.md]: Created the link.

File: charts/SDFactory/.helmignore [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+*.swp
+*.bak
+*.tmp
+*.orig
+*~
+# Various IDEs
+.project
+.idea/
+*.tmproj
+.vscode/
```
[the remark this answers is not in the document]
Comment (adkumar1) [reply charts/SDFactory/.helmignore]: Changed the content

File: CHANGELOG.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (carslen) [reply CHANGELOG.md]: Better, but I'm still missing the Added/Changed/Fixed/... sections for 1.2.0.

File: CHANGELOG.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (carslen) [reply CHANGELOG.md:15]: Better, but I'm still missing the Added/Changed/Fixed/... sections for 1.2.1.

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (carslen) [reply README.md]: Perfect. Please remove the ":-"  in the headline

**Comment by carslen** [comment]
Please add the missing `charts/SDFactory/Readme.md` file (use helm-docs to generate the file and add valuable comments to `charts/SDFactory/values.yaml`). 

If this file has been added (incl. the comments in `values.yaml`) and the few open remarks got solved, then we're good to go.

File: CHANGELOG.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (adkumar1) [reply CHANGELOG.md:15]: Done! Segregated the added, changed

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (adkumar1) [reply README.md]: Removed

**Review by carslen — APPROVED** [review]

**Comment by carslen** [comment]
I'll merge this now, although GitGuardian complains. Found creds where example ones and removed

**Pull request merged**